src/assets/database.sql

```diff
@@ -341,6 +341,33 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Time sec', 'Seconds to delay the DBMS response', 'sqlmap --time-sec 5', 26, 'input', '--time-sec', '5'),
 ('Techniques', 'SQL injection techniques to use', 'sqlmap --technique BEUSTQ', 26, 'input', '--technique', 'BEUSTQ');
 
+--
+-- Populate data for `commands`: dnsenum
+--
+
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('Set DNS server', 'Use this DNS server for A, NS and MX queries.', 'dnsenum --dnsserver 10.0.0.1 example.com', 14, 'input', '--dnsserver', NULL, NULL, 'GENERAL OPTIONS'),
```

Comment:
It would be nice to fill `example` field for this parameter as well for all those `input` types.